### PR TITLE
Updates styles on needs page to center header and filters

### PR DESCRIPTION
**Pull request by dcalacci**
- standardizes mobile media query to 411 px across some components
- switches flex direction (column -> row) for mobile on header and filter list
- fixes width of filter buttons on mobile so it appears in an orderly column
- wraps all content if too large for screen using flex-wrap

addresses #21

### CODE REVIEW

File: src/components/Header/styles.module.css [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
+  flex-direction: row;
+  flex-wrap: wrap;
   align-items: center;
   justify-content: space-between;
   margin: 1rem;
   padding: 2rem 4rem;
   text-align: center;
 }
 
+@media screen and (max-width: 411px) {
```
Comment (esetnik) [inline src/components/Header/styles.module.css:13]: @dcalacci I'm curious. Where does the magic 411px number come from?

Comment (dcalacci) [reply src/components/Header/styles.module.css:13]: should actually change all to be 768 (tablet and below). there was a media query at 410, but 411 supports i.e. pixel 2 xl, larger portrait phones.

**Comment by dcalacci** [comment]
regarding wrapping -- How do we want all these to be displayed on desktop and up? One simple way is to just fix button size and make it a grid under search.

**Comment by esetnik** [comment]
> regarding wrapping -- How do we want all these to be displayed on desktop and up? One simple way is to just fix button size and make it a grid under search.

Makes sense to me. I actually think the new request action probably belongs on the right and the filters on the left. Not sure why but it makes more sense to me to have the majority of the controls left aligned. Also the button should be a different color or in some way indicate that it is different from the filter buttons.

**Comment by dcalacci** [comment]
just added a grid + flexbox fallback situation. looking at the commit its clear i'm not following whatever style guidelines are there (my editor has its own defaults) so happy to change if there's a guide somewhere

File: src/components/Header/index.js [hunk 2]
```diff
@@ -4,15 +4,17 @@ import cs from "./styles.module.css"
 
 export default () => {
   return (
-    <header className={cs.header}>
-      <a href="https://mutualaidmamas.com/">
+      <header className={cs.header}>
+          <div className={cs.logo}>
```
Comment (esetnik) [inline src/components/Header/index.js:8]: @dcalacci what editor are you using? We use [prettier](https://prettier.io/) for formatting. This looks very strange. with the enclosing `<div>` being indented more than its children??

File: src/components/Header/styles.module.css [hunk 3]
```diff
@@ -1,27 +1,49 @@
...
-  border: 4px solid #253f58;
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  align-items: center;
-  justify-content: space-between;
-  margin: 1rem;
-  padding: 2rem 4rem;
-  text-align: center;
+    display: flex;
```
Comment (esetnik) [inline src/components/Header/styles.module.css:2]: Looks like it converted to 4 spaces. Please use prettier for formatting. If you can't get your editor to work with prettier you can use the prettier cli to format the files before committing them. See `prettier --write` https://prettier.io/docs/en/cli.html

**Comment by esetnik** [comment]
> just added a grid + flexbox fallback situation. looking at the commit its clear i'm not following whatever style guidelines are there (my editor has its own defaults) so happy to change if there's a guide somewhere

Please use prettier. See https://github.com/mmoutenot/mamas-needs-offers/blob/master/.prettierrc for our ruleset.

File: src/components/Header/styles.module.css (hunk 3 above)
Comment (dcalacci) [reply src/components/Header/styles.module.css]: works fine, just configured. lots of other files in the repo are off-guidelines I noticed fyi!

File: src/components/Header/index.js (hunk 2 above)
Comment (dcalacci) [reply src/components/Header/index.js]: lol I'm in the middle of configuring a new machine. I use emacs and not all of my dev environment is set up yet, that was.... not automatically done, lol

File: src/components/Header/styles.module.css (hunk 3 above)
Comment (esetnik) [reply src/components/Header/styles.module.css]: Not sure if I understand. Can you point to an example?

**Comment by dcalacci** [comment]
sorry about the mess. should be up to date. here's how it looks. 

## desktop
Screen Shot 2020-04-16 at 2 37 48 PM

## tablet:
Screen Shot 2020-04-16 at 4 07 31 PM

## mobile (wide)
Screen Shot 2020-04-16 at 4 07 47 PM

## mobile
Screen Shot 2020-04-16 at 4 09 57 PM

File: src/components/Header/index.js (hunk 2 above)
Comment (esetnik) [reply src/components/Header/index.js]: Emacs is hardcore.

File: src/components/Header/styles.module.css (hunk 3 above)
Comment (dcalacci) [reply src/components/Header/styles.module.css]: well I ran `prettier --write` from root with the .prettierrc from the repo and it updated a whole bunch of files that I haven't edited: 

Screen Shot 2020-04-16 at 4 26 20 PM

**Comment by esetnik** [comment]
Looks good to me. Definitely better than what we have now. @mewtaylor any feedback you want to add? If anything I think the search bar belongs above the other filters rather than with the new button because it’s kind of a filter.

**Comment by esetnik** [comment]
@dcalacci do you mind rebasing this when you get a chance?

**Pull request merged**